tools/CouchDB_Confirm_Integrity.php

```diff
@@ -88,22 +88,24 @@ function run()
                 FROM candidate c
                 LEFT JOIN session s USING (CandID)
                 WHERE c.PSCID=:PID AND s.Visit_label=:VL",
-                array(
+                [
                     "PID" => $pscid,
-                    "VL" => $vl
-                )
+                    "VL"  => $vl,
+                ]
             );
 
             if (!empty($sqlDB) && $sqlDB['cActive'] == 'N') {
                 print "PSCID $pscid is inactive but $row[id] still exists. 
                 Deleting Doc.\n";
 
                 $this->CouchDB->deleteDoc($row['id']);
-            } else if (!empty($sqlDB) && $sqlDB['Active'] != 'Y') {
+            } elseif (!empty($sqlDB) && $sqlDB['Active'] != 'Y') {
```

Comment:
What rule was complaining about this?

Reply:
@driusan Nothing seems to justify this change. I reverted it.